macros/sfdc_formula_view.sql

```diff
@@ -7,13 +7,17 @@
     )
 }}
 
+-- Raise a compiler error if users are trying to use full_statement_version=false. We are keeping the variable in the macro, however, since we don't want errors if they previously set it to true.
+{% if not full_statement_version %}
+{{ exceptions.raise_compiler_error("\ERROR: Parameter full_statement_version=false is no longer supported. Update your " ~ this.identifier|upper ~ " model to remove the full_statement_version parameter.\n") }}
```

Comment:
Using the `exceptions.raise_compiler_error` feels dangerous here. This means if the `full_statement_version=false` then the entire dbt project will fail with the compile error. For example, in my project I have a model that is completely unrelated to my formula_field model, but it refuses to compile because of the compile failure log.

![image](https://github.com/fivetran/dbt_salesforce_formula_utils/assets/74217849/6fb04cac-1bec-4718-9177-27286fff6557)

I think we should definitely raise the error in this model. However, I don't advise we make this result in the entire dbt project failing.

Reply:
Can we explore possible alternatives.

Reply:
Updated with the floating string method from our discussion earlier this week!